File: sumSubMult.js

const readline = require('readline').createInterface({
  input: process.stdin,
  output: process.stdout
});
readline.question('Ingresa un numero: ',n1=> {
  readline.question('Ingresa un segundo numero: ',n2 => {
    let n = parseInt(n1), n3= parseInt(n2);
    console.log(`La suma es: ${n+n3},  la resta es: ${n-n3}, la multiplicacion es ${n*n3}, y la division es ${n/n3}`)
    readline.close();
  });
});

// let sum = n1+n2,sub=n1-n2,mult = n1*n2,div = n1/n2;
// n3.forEach(el => {
//   sum += el;
//   mult*=el;
//   sub-=el;
//   div/=el;
// });

File: sumSubMult.c
#include<stdio.h>
int main(){
  int n1, n2;
  printf("Introduzca un numero entero: ");
  scanf("%d", &n1);
  printf("Introduzca un segundo entero: ");
  scanf("%d", &n2);
  printf("La suma de los dos numeros es: %d, la resta de los dos numeros es: %d, la multiplicacion de los dos numeros es: %d y la division de los dos numeros es: %d\n",n1+n2, n1-n2, n1*n2,n1/n2);
  return 0;
}